pages/general-information-and-resources/tech-policies/tts-pages.md

```diff
@@ -0,0 +1,84 @@
+---
+title: TTS Pages
+questions:
+  - tts-tech-operations
+  - cg-pages
+redirect_from:
+  - /tts-pages/
+---
+# TTS Pages - Authority to Use (ATU) Process
+
+>This guide is intended for `Website Managers` to meet their requirements under the TTS Pages Authority to Operate (ATO)
+
+"TTS Pages" is GSA's Authority to Operate (ATO) using [Cloud.gov's FEDRAMP Authorization](https://marketplace.fedramp.gov/products/F1607067912) for their [Cloud.gov Pages](https://pages.cloud.gov) service. As such, it adds the Security Controls around the Source Code and Contents for the Website (e.g. Github). It provides TTS users with a fast and secure approach to getting a Web Presence for your projects/programs. This page defines the requirements for launching a new website at GSA using [Cloud.gov Pages](https://pages.cloud.gov)
+
+>Follow this link for more information about using [Cloud.gov Pages](tools/pages)
+
+## Launching a New Static Website at TTS
+
+- [ ] Determine who will serve as the `Website Manager`
+
+>`Website Manager` is defined by the [GSA's Digital.gov Program](https://digital.gov/2023/03/24/who-is-your-website-manager/) 
+
+- [ ] Turn on all [Github Advanced Security](https://docs.github.com/en/get-started/learning-about-github/about-github-advanced-security) for the Site's Repository and `Enable`:
+    - [ ] [CodeQL](https://docs.github.com/en/code-security/code-scanning)
+    - [ ] [Dependabot](https://docs.github.com/en/code-security/dependabot/)
+    - [ ] Secret Scanning
+- [ ] Designate (One) Github Team with `Admin` Access to the repository
+- [ ] Designate (One or More) Github Team(s) with `Write` Access to the repository
+- [ ] Submit a Pull Request to add the Github Repository for the Website to [GSA](https://github.com/GSA/.allstar/blob/main/allstar.yaml) or [GSA-TTS](https://github.com/GSA-TTS/.allstar/blob/main/allstar.yaml) .allstar Github Configuration Scanner)
+- [ ] Create a [`SECURITY.md` file](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository)
+- [ ] Follow GSA's [Digital Lifecycle Program](https://insite.gsa.gov/employee-resources/communications/websites/strategy-policy-and-standards/digital-lifecycle-program?term=DLP) for the Website
+- [ ] Confirm your Website on [Touchpoint's Registry](https://touchpoints.app.cloud.gov/admin/websites) or Submit a new website request in [Touchpoints](https://touchpoints.app.cloud.gov/admin/websites/new)
+
+## Authority to Use (ATU) Review
+Submit an [Authority to Use (ATU) Request](https://github.com/GSA-TTS/tts-pages/issues/new?)
+
+- [ ] Resolve any Critical or High security findings from Security Scanners
+- [ ] Resolve any Web Scan findings provided by GSA Security Team
+
+Once the review is completed the Site Manager will be sent their ATU Approval package via Docusign. The Site Manager will be responsible for managing Security Findings over the lifecycle of the Website.
```

Comment:
comma needed after "completed". Also not sure about all of these capitalized nouns.